### ISSUE TITLE
Formie dashboard widget server error when using date range.

### ISSUE THREAD

**Issue by JohnCBent** [verbb/formie#461]
**Description**
We're encountering a server error when using the recent form submissions widget. If we select line graph view, and use a date range to filter submissions, then save, Craft returns a server error message. 
When we reload the dashboard we see a yii exception page with this error stack.

```
2021-05-06 12:27:07 [-][1][up3ukok6r5n5d94gsodmv5tfem][error][TypeError] TypeError: date_diff(): Argument #1 ($baseObject) must be of type DateTimeInterface, string given in /Users/john/Desktop/Craft/washtrust/vendor/verbb/formie/src/widgets/RecentSubmissions.php:373
Stack trace:
#0 /Users/john/Desktop/Craft/washtrust/vendor/verbb/formie/src/widgets/RecentSubmissions.php(373): date_diff('1617508800', '1620360000')
#1 /Users/john/Desktop/Craft/washtrust/vendor/verbb/formie/src/widgets/RecentSubmissions.php(335): verbb\formie\widgets\RecentSubmissions->getDateRangeInterval()
#2 /Users/john/Desktop/Craft/washtrust/vendor/verbb/formie/src/widgets/RecentSubmissions.php(176): verbb\formie\widgets\RecentSubmissions->_createChartQuery(Object(craft\db\Query), Array, Array)
#3 /Users/john/Desktop/Craft/washtrust/vendor/craftcms/cms/src/controllers/DashboardController.php(508): verbb\formie\widgets\RecentSubmissions->getBodyHtml()
#4 /Users/john/Desktop/Craft/washtrust/vendor/craftcms/cms/src/controllers/DashboardController.php(107): craft\controllers\DashboardController->_getWidgetInfo(Object(verbb\formie\widgets\RecentSubmissions))
#5 [internal function]: craft\controllers\DashboardController->actionIndex()
#6 /Users/john/Desktop/Craft/washtrust/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#7 /Users/john/Desktop/Craft/washtrust/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#8 /Users/john/Desktop/Craft/washtrust/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction('index', Array)
#9 /Users/john/Desktop/Craft/washtrust/vendor/yiisoft/yii2/base/Module.php(534): craft\web\Controller->runAction('index', Array)
#10 /Users/john/Desktop/Craft/washtrust/vendor/craftcms/cms/src/web/Application.php(274): yii\base\Module->runAction('dashboard/index', Array)
#11 /Users/john/Desktop/Craft/washtrust/vendor/yiisoft/yii2/web/Application.php(104): craft\web\Application->runAction('dashboard/index', Array)
#12 /Users/john/Desktop/Craft/washtrust/vendor/craftcms/cms/src/web/Application.php(259): yii\web\Application->handleRequest(Object(craft\web\Request))
#13 /Users/john/Desktop/Craft/washtrust/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#14 /Users/john/Desktop/Craft/washtrust/web/index.php(22): yii\base\Application->run()
#15 /Users/john/.composer/vendor/laravel/valet/server.php(219): require('/Users/john/Des...')
#16 {main}
```

**Steps to reproduce**
1.Edit recent submissions dashboard widget
2. Select line graph view
3. Enter a date range
4. Save widget settings.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: N/A
- Client-side Validation: N/A
- Custom Form Templates: N/A

**Additional info**
- Plugin version: 1.4.3
- Craft version: 3.6.12
- Multi-site: Yes 

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.4](https://github.com/verbb/formie/releases/tag/1.4.4)
